extensions/sat/README.md

```diff
@@ -11,24 +11,35 @@ This document explains the fields of the Satellite Extension to a STAC Item. Sat
 The Satellite extension requires the [Instrument Fields](../../item-spec/common-metadata.md#instrument).
 
 - [Example (Landsat 8)](examples/example-landsat8.json)
+- [Example (Sentinel 1)](examples/example-sentinel1.json)
 - [JSON Schema](json-schema/schema.json)
 
 ## Item fields
 
 | Field Name       | Type                     | Description |
 | ---------------- | ------------------------ | ----------- |
+| sat:platform_international_designator | string | The International Designator, also known as COSPAR ID, and NSSDCA ID |
 | sat:orbit_state        | string        | The state of the orbit. Either `ascending` or `descending` for polar orbiting satellites, or `geostationary` for geosynchronous satellites |
+| sat:absolute_orbit     | integer       | The obsolute orbit number at the time of acquisition. |
 | sat:relative_orbit     | integer       | The relative orbit number at the time of acquisition. |
+| sat:anx_date_time     | date_time    | The Ascending Node Crossing (ANX) UTC time |
 
 *At least one of the fields must be specified.*
 
+**sat:platform_international_designator** is the [International Designator](https://en.wikipedia.org/wiki/International_Designator), also known as COSPAR ID, and NSSDCA ID and is an international identifier assigned to artificial objects in space.
+
 **sat:orbit_state** indicates the type and current state of orbit. Satellites are either geosynchronous in which case they have one state: `geostationary`, or they are sun synchronous (i.e., polar orbiting satellites) in which case they are either `ascending` or `descending`. For sun synchronous satellites it is daytime during one of these states, and nighttime during the other.
 
+**sat:absolute_orbit** is a count of orbits from 1 to the number of orbits made in the total satellite lifecycle
+
 **sat:relative_orbit** is a count of orbits from 1 to the number of orbits contained in a repeat cycle, where relative orbit 1 starts from a specific reference location of the sub-satellite point (the point on the earth directly below the satellite). It resets to 1 when the sub-satellite point revisits the reference location.
 
+**sat:anx_date_time** is the UTC time when the satellite crosses the [Ascending Node](https://en.wikipedia.org/wiki/Orbital_node). For geocentric and heliocentric orbits, the ascending node (or north node) is where the orbiting object moves north through the plane of reference.
+
```

Comment:
```suggestion
```suggestion
Used to quickly compute orbital parameters without having to download the product. For instance to compute on the fly a baseline between 2 satellite acquisition, to find the best candidate in the archive from a post-disaster event (e.g earthquake) scene acquisition for a DInSAR processing.
```
```